cmd/signozschemamigrator/migrate.go

```diff
@@ -72,7 +73,22 @@ func main() {
 
 	// set cluster env so that golang-migrate can use it
 	// the value of this env would replace all occurences of {{.SIGNOZ_CLUSTER}} in the migration files
+	// TODO: remove this log after dirtry migration issue is fixed
+	logger.Info("Setting env var SIGNOZ_CLUSTER", zap.String("cluster-name", clusterName))
 	os.Setenv("SIGNOZ_CLUSTER", clusterName)
+	// TODO: remove this section after dirtry migration issue is fixed
+	clusterNameFromEnv := ""
+	for _, kvp := range os.Environ() {
+		kvParts := strings.SplitN(kvp, "=", 2)
+		if kvParts[0] == "SIGNOZ_CLUSTER" {
+			clusterNameFromEnv = kvParts[1]
+			break
+		}
+	}
+	if clusterName == "" {
+		logger.Fatal("Failed to set env var SIGNOZ_CLUSTER")
```

Comment:
`os.Setenv` returns an error when it fails to set or something else happens. It is also the most reliable info to check if the call to `Setenv` was successful.

Reply:
Since we are using the fork of the migrate and can make any changes, adding some verbose logs close to the template exec in the SigNoz/golang-migrate is better.

Reply:
Is there a reason why this runs in go routine https://github.com/SigNoz/golang-migrate/blob/7e92866dfed375fc6f0a9190d7b2324f8f6cbdc3/template.go#L41-L44 and return errors are ignored?

Reply:
Not sure @srikanthccv, this is the library's implementation

Reply:
How's that a library implementation? We added the template part.

Reply:
I'm sorry i didn't phrase it properly..it is a library fork's implementation which we are using

Reply:
> Since we are using the fork of the migrate and can make any changes, adding some verbose logs close to the template exec in the SigNoz/golang-migrate is better.

I have added some logs to near exec in the migrate fork too @srikanthccv 
i'm also keeping the logs here in migrate.go for now..it would be duplicate logs but we can remove them once this issue is figured out